index.js

```diff
@@ -2,11 +2,14 @@ import { ApolloServer } from 'apollo-server'
 import { readFileSync } from 'fs'
 import resolvers from './resolvers'
 
-const typeDefs = readFileSync('./schemas/schema.graphql').toString('utf-8')
+const queries = readFileSync('./schemas/schema.graphql').toString('utf-8')
+// const mutations = readFileSync('./schemas/mutations.graphql').toString('utf-8')
+const objects = readFileSync('./schemas/objects.graphql').toString('utf-8')
+// const inputs = readFileSync('./schemas/inputs.graphql').toString('utf-8')
```

Comment:
Suggestion:
use `fs.readdirSync` with wildcard `*.graphql` to get an array of `filepaths`
use `filepaths.map(()=>...)` to create a typeDefs